### ISSUE TITLE
Add statesmen to the forum phase

### ISSUE THREAD

**Issue by iamlogand** [iamlogand/republic-of-rome-online#228]
Requires #225, makes sense to do this after #227.

During the forum phase, each faction will be able to pick up a statesman from the "deck". Unplayed statesmen will use a model called `Concession`. As they are not playable within the scope of this issue, there is no need to worry about their specific attributes.

**Comment by iamlogand** [comment]
Closing to merge into issue #227
